src/tests/ftest/pool/evict_test.py

```diff
@@ -33,8 +33,7 @@ def connected_pool(self, hostlist, targets=None):
             TestPool: a TestPool objectfor the created pool
 
         """
-        pool = TestPool(self.context, self.get_dmg_command())
-        pool.get_params(self)
+        pool = self.get_pool(create=False)
```

Comment:
This should either be `self.add_pool(create=False)` or `self.pool.append(self.get_pool(create=False))` as https://github.com/daos-stack/daos/blob/master/src/tests/ftest/pool/evict_test.py#L50-L51 could send us into `tearDown()` before either https://github.com/daos-stack/daos/blob/master/src/tests/ftest/pool/evict_test.py#L84 or https://github.com/daos-stack/daos/blob/master/src/tests/ftest/pool/evict_test.py#L165 can assign/append `self.pool`.  Since this method is used by `test_evict_bad_uuid/evict_badparam` - where `self.pool` is a single value - and `test_evict` - where `self.pool` is a list, possibly the simplest solution would be convert `evict_badparam` to use a single value `self.pool` list and than have this method use `self.pool.append(self.get_pool(create=False))`.